lib/instrumentation/fastify/spec-builders.js

```diff
@@ -75,6 +75,12 @@ function buildMiddlewareSpecForRouteHandler(shim, path) {
      * The isFinal param determines whether or not a path is appended for
      * this particular piece of middleware. (i.e. if this is the final handler
      * that is actually handling the request, the path is actually left on)
+     *
+     * @param shim
```

Comment:
I know this is unrelated but personally I'm not a fan of requiring specifying these in jsdoc without types